<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVCameraDouble</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblImageView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCapture">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPS">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResolution">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>560</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxResolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>530</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPS2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>530</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS: 0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxResolution2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>530</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCam2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCapture2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>530</y>
      <width>100</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResolution2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>560</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImageView2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFPS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFPS2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>590</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetCamProp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Prop</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPropID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPropVal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Property ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Property Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>630</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Property ID:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPropID2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>650</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPropVal2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>630</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Property Value:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetCamProp2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>630</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Prop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
